marlowe/test/test-report.md

```diff
@@ -400,7 +400,7 @@ Several other property-based tests, in the [`Spec.Marlowe.Marlowe`](src/Spec/Mar
 
 ## Test Oracle for Marlowe
```

Comment:
I would rename this for Test Spec, as oracle can be misinterpreted

Reply:
Fixed in c4cbff6a21bc2da875aa867375658d8f8fe803a6.